test/textformatter.cpp

```diff
@@ -83,9 +83,10 @@ TEST_CASE("textformatter: wrappable sequences longer then format width are force
 
 TEST_CASE("textformatter: when wrapping, spaces at the beginning of lines are dropped") {
 	textformatter fmt;
-	fmt.add_line(wrappable, "just a test");
+	fmt.add_line(wrappable, "   just a test");
 
 	const std::string expected =
+		" \n"
```

Comment:
Hmm, I don't think this is correct. "Drop" means "get rid of", not "squash into one and put onto separate line".

There's two solutions:
1. I can decline this PR (thanks nonetheless!) and file an issue about `textformatter` (most probably immediately fixing this because it looks easy);
2. if you want, you can fix `textformatter` to really drop that spaces, update this PR and I'll merge it.

Which one do you like best?